pkg/executor/importer/chunk_process.go

```diff
@@ -227,20 +231,20 @@ func newChunkEncoder(
 	offset int64,
 	sendFn func(ctx context.Context, batch *encodedKVGroupBatch) error,
 	collector execute.Collector,
-	logger *zap.Logger,
 	encoder *TableKVEncoder,
 	keyspace []byte,
 ) *chunkEncoder {
 	return &chunkEncoder{
-		chunkName:     chunkName,
-		readFn:        readFn,
-		offset:        offset,
-		sendFn:        sendFn,
-		collector:     collector,
-		logger:        logger,
-		encoder:       encoder,
-		keyspace:      keyspace,
-		groupChecksum: verify.NewKVGroupChecksumWithKeyspace(keyspace),
+		chunkName:        chunkName,
+		readFn:           readFn,
+		offset:           offset,
+		sendFn:           sendFn,
+		collector:        collector,
+		encoder:          encoder,
+		keyspace:         keyspace,
+		minDeliverBytes:  MinDeliverBytes,
```

Comment:
Maybe rename to `DefaultMinDeliverBytes`
```suggestion
		minDeliverBytes:  DefaultMinDeliverBytes,
```